content/zh/docs/contribute/_index.md

```diff
@@ -170,4 +170,10 @@ documentation, but it should help you get started.
   - 对文档测试提出改进建议
   - 对 Kubernetes 网站或其他工具提出改进建议
 
+
```

Comment:
advice:don't delete origin english
it is easy to review to  reviewer.

Reply:
OK, thx